### PR TITLE
#169314391 refactor Article(s) and rest password 

### PR DESCRIPTION
**Pull request by salviosage**
#### What does this PR do?

- I should get authors info, rating  average, and claps  when I fetch the article 

- I could have a link to reset password  setted to `/resetpassword` instead of `/reset`

#### Description of Task to be completed?

- refactoring article returning data

#### How should this be manually tested?

- by cloning this repo without to this branch  and start the server,

- after that try getting all articles or a single article 

#### What are the relevant pivotal tracker stories?
[#169314391](https://www.pivotaltracker.com/story/show/169314391)

#### Screenshots (if appropriate)
image

### CODE REVIEW

**Review by habinezadalvan — APPROVED** [review]
LGTM

File: src/controllers/articles.controller.js [hunk 1]
```diff
@@ -328,16 +354,12 @@ class Articles {
       where: { slug: req.params.slug }
     });
     if (!findArticle) {
-      return res
-        .status(200)
-        .json({ status: 200, message: 'That article does not exist' });
+      util.setError(404, 'That article does not exist!');
+      return util.send(res);
```
Comment (Anguandia) [inline src/controllers/articles.controller.js:358]: Thats brilliant! I was going to raise that!

File: src/controllers/articles.controller.js [hunk 2]
```diff
@@ -347,10 +369,8 @@ class Articles {
       description,
       taglist: req.body.taglist.split(' ')
     });
-    return res.status(200).json({
-      status: 200,
-      updatedArticle
-    });
+    util.setSuccess(200, 'Article Updated successfully!', updatedArticle);
```
Comment (Anguandia) [inline src/controllers/articles.controller.js:372]: And this too! thanks alot

File: src/controllers/user.controller.js [hunk 3]
```diff
@@ -412,7 +412,7 @@ class UserController {
 
       const token = await Helper.generateToken(payload, (60 * 60));
       // create password reset link
-      const resetUrl = `${process.env.FRONT_END_URL}/reset?token=${token}`;
+      const resetUrl = `${process.env.FRONT_END_URL}/resetpassword?token=${token}`;
```
Comment (Anguandia) [inline src/controllers/user.controller.js:415]: Goodwork

**Review by Anguandia — APPROVED** [review]

**Comment by Anguandia** [comment]
My concerns are all covered, thanks!

**Review by nkalyesubula — APPROVED** [review]
LGTM

**Pull request merged**